src/FlubuCore/Tasks/Process/RunProgramTask.cs

```diff
@@ -250,7 +250,14 @@ protected override int DoExecute(ITaskContextInternal context)
 
             DoLogInfo($"Running program from '{workingFolder}':");
 
-            DoLogInfo($"{command.CommandName} {commandArgs}{Environment.NewLine}", Color.DarkCyan);
+            if (command.CommandName.Contains("cmd.exe"))
```

Comment:
I've tested with `npm` and `choco`, and this is how I try to satisfy both cases while maintaining command arguments masks work.

Reply:
Maybe for loging purposes changing from:

``` c#   
 if (command.CommandName != null && command.CommandName.Contains("cmd.exe"))
 {
      DoLogInfo($"{cmd}{commandArgs}{Environment.NewLine}", Color.DarkCyan);
 }
else
 {
      DoLogInfo($"{command.CommandName}{commandArgs}{Environment.NewLine}", Color.DarkCyan);
 }
```
 to 
``` c#   
DoLogInfo($"{cmd}{commandArgs}{Environment.NewLine}", Color.DarkCyan);
```
What do you think? Not sure if we would lose any important info about the command name in the logs. I think it should be ok for all scenarios.

Reply:
Yes I think that would be OK. I assume few guys would care to know whether the program is executed via cmd.exe or not. Thanks for the suggestion!